lib/Text_Wiki/Text/Wiki/Parse/Default/Deflist.php

```diff
@@ -54,7 +54,11 @@ class Text_Wiki_Parse_Deflist extends Text_Wiki_Parse {
     *
     */
 
-    public $regex = '/\n((: ).*\n)(?!(: |\n))/Us';
+    public $regex = '/
+        \n
+        ((:\s).*?\n)   # Match colon, whitespace, then text up to newline
+        (?!(:\s|\n))   # Do not match if followed by colon+ws or newline
+        /sx';
```

Comment:
Do we want to lose the U (ungreedy) flag here?

Reply:
For sure - it'll be a pain to debug down the line when someone inevitably forgets to look at the modifiers. I've added a `?` to the `.*` to make it ungreedy. U is poison to consistency.